code/client/src/core/hooks/vehicle.cpp

```diff
@@ -129,22 +130,23 @@ void C_Human2CarWrapper__EndDrive(SDK::C_Human2CarWrapper *pThis, SDK::C_Actor *
 
 static InitFunction init([]() {
     // Disable automated vehicle enable (engine, siren, beacon lights etc...) when player enter it
-    const auto C_Human2CarWrapper__StartDrive_Addr = hook::pattern("48 89 5C 24 ? 48 89 6C 24 ? 56 57 41 57 48 83 EC 70 45 33 FF").get_first();
+    const auto C_Human2CarWrapper__StartDrive_Addr = hook::get_pattern("48 89 5C 24 ? 48 89 6C 24 ? 56 57 41 57 48 83 EC 70 45 33 FF");
     MH_CreateHook((LPVOID)C_Human2CarWrapper__StartDrive_Addr, (PBYTE)C_Human2CarWrapper__StartDrive, reinterpret_cast<void **>(&C_Human2CarWrapper__StartDrive_original));
 
     // Hook the game EndDrive method so we can avoid it disabling the vehicle (engine, lights etc...)
     const auto C_Human2CarWrapper__EndDrive_Addr = hook::get_opcode_address("E8 ? ? ? ? 48 8B 43 08 48 8B 7C 24 ?");
     MH_CreateHook((LPVOID)C_Human2CarWrapper__EndDrive_Addr, (PBYTE)C_Human2CarWrapper__EndDrive, reinterpret_cast<void **>(&C_Human2CarWrapper__EndDrive_original));
 
-    const auto C_CarActionEnter__TestActionInternal_Addr = hook::pattern("40 53 55 57 41 54 41 55 48 83 EC 40 41 0F B6 F8").get_first();
+    const auto C_CarActionEnter__TestActionInternal_Addr = hook::get_pattern("40 53 55 57 41 54 41 55 48 83 EC 40 41 0F B6 F8");
     MH_CreateHook((LPVOID)C_CarActionEnter__TestActionInternal_Addr, (PBYTE)C_CarActionEnter__TestActionInternal, reinterpret_cast<void **>(&C_CarActionEnter__TestActionInternal_original));
 
-    const auto C_CarActionBreakIn__TestActionBreakIn_Addr = hook::pattern("48 89 5C 24 ? 48 89 6C 24 ? 48 89 74 24 ? 57 41 56 41 57 48 83 EC 30 45 0F B6 F8").get_first();
+    const auto C_CarActionBreakIn__TestActionBreakIn_Addr = hook::get_pattern("48 89 5C 24 ? 48 89 6C 24 ? 48 89 74 24 ? 57 41 56 41 57 48 83 EC 30 45 0F B6 F8");
     MH_CreateHook((LPVOID)C_CarActionBreakIn__TestActionBreakIn_Addr, (PBYTE)C_CarActionBreakIn__TestActionInternal, reinterpret_cast<void **>(&C_CarActionBreakIn__TestActionInternal_original));
 
-    const auto C_CarActionLeave__TestAction_Addr = hook::pattern("40 53 48 83 EC 20 48 8B DA E8 ? ? ? ? 48 8B C8 4C 8B 00 41 FF 90 ? ? ? ? 48 3B D8 75 29 E8 ? ? ? ? 48 8B C8 48 8B 10 FF 92 ? ? ? ? 48 8B C8 BA ? ? ? ? E8 ? ? ? ? 84 C0 0F 94 C0 48 83 C4 20 5B C3 B0 01 48 83 C4 20 5B C3 CC CC CC CC CC CC CC CC CC CC CC CC CC CC CC 41 B0 01 E9 ? ? ? ? CC CC CC CC CC CC CC CC 48 89 5C 24 ? 48 89 6C 24 ?").get_first();
+    const auto C_CarActionLeave__TestAction_Addr = hook::get_pattern("40 53 48 83 EC 20 48 8B DA E8 ? ? ? ? 48 8B C8 4C 8B 00 41 FF 90 ? ? ? ? 48 3B D8 75 29 E8 ? ? ? ? 48 8B C8 48 8B 10 FF 92 ? ? ? ? 48 8B C8 BA ? ? ? ? E8 ? ? ? ? 84 C0 0F 94 C0 48 83 C4 20 5B C3 B0 01 48 83 "
+                                                                     "C4 20 5B C3 CC CC CC CC CC CC CC CC CC CC CC CC CC CC CC 41 B0 01 E9 ? ? ? ? CC CC CC CC CC CC CC CC 48 89 5C 24 ? 48 89 6C 24 ?");
```

Comment:
This shouldn't go on multiple lines with string interpolation

Reply:
Can't, this modification is made by the code formatter

Reply:
C/C++ will concat to single line anyways, this should we fine, just ugly.

Reply:
Ugly is the point :D Lets fix formatter!

Reply:
I'm supposed to do it? 
I admit that I am not expert enough to control the impacts...